Specifications/QIR/Callables.md

```diff
@@ -318,6 +312,15 @@ language-specific compiler.
 The lambda body may need to include additional code that performs argument tuple
 construction before calling the underlying callable.
 
+#### Memory Management Table
+
+Since any captured values need to remain alive as long as the callable value exists, they also need to be unreferenced when the callable value is released. While sufficient type information for the captured values is known upon creation of the value, the information is no longer available at the time when it is released.
+Upon creation, a table with two function pointers for modifying reference and alias counts for captured values is hence associated with a callable value. 
+
+Like the implementation table, the table is defined as global constant with a unique name. It contains two pointers of type `void(%Tuple*, i64)*`; the first one points to the function for modifying the reference counts of captured values, the second points to the one for modifying the alias counts. Either of those pointers may be null, and if no values were captured, a null pointer should be passed instead of a table upon callable creation.
```

Comment:
```suggestion
Like the implementation table, the table is defined as global constant with a unique name. It contains two pointers of type `void(%Tuple*, i64)*`; the first one points to the function for modifying the reference counts of captured values, the second points to the one for modifying the alias counts. Either of those pointers may be null, and if no managed values were captured, a null pointer should be passed instead of a table upon callable creation.
```

I added "managed" because a null pointer should be also provided if all that's captured are some integers or other scalars.